apps/costcontrol/js/settings/settings.js

```diff
@@ -128,8 +135,12 @@ var Settings = (function() {
         closeSettings();
       });
 
-      function _setResetTimeToDefault(value, old, key, settings) {
-        var today = new Date();
+      function _setDataPeriodBounds(value, old, key, settings) {
+        var today = InputParser.exportDate(new Date());
```

Comment:
I don't understand this part. Why are we setting the `startingDate` always to `today`?